src/pages/workspaces/workspace/Analyses.js

```diff
@@ -328,6 +338,16 @@ const Analyses = _.flow(
     ])
   ])
 
+  const activeFileTransferMessage = div({
+    style: _.merge(
+      Style.elements.card.container,
+      { backgroundColor: colors.warning(0.15), flexDirection: 'none', justifyContent: 'start', alignItems: 'center' })
+  }, [
+    icon('warning-standard', { size: 19, style: { color: colors.warning(), flex: 'none', marginRight: '1rem' } }),
+    'Copying 1 or more notebooks from another workspace.',
```

Comment:
we actually have more artifacts now..it might `.Rmd` file and `.R` files in the future, maybe use some other more generic term?

Reply:
Would `Copying 1 or more analysis files from another workspace.` work? I am not sure what the best generic term is for this but am open to suggestions!

Reply:
ah, I like that 👍

Reply:
When the context isn't obvious and verbosity isn't a major concern, I prefer to use the term `interactive analysis` since `batch analysis` is also supported by Terra but I don't feel strongly here.